ui/decredmaterial/switch.go

```diff
@@ -3,16 +3,26 @@
 package decredmaterial
 
 import (
+	"image"
 	"image/color"
 
+	"gioui.org/f32"
+	// "gioui.org/internal/f32color"
+	"gioui.org/io/pointer"
 	"gioui.org/layout"
+	"gioui.org/op"
+	"gioui.org/op/clip"
+	"gioui.org/op/paint"
 	"gioui.org/unit"
 	"gioui.org/widget"
-	"gioui.org/widget/material"
+	// "gioui.org/widget/material"
 )
 
 type Switch struct {
```

Comment:
This widget needs a `Changed()` event for when the enabled state of the switch is changed.